### PR TITLE
just added some QoL for foraging!

### PR DESCRIPTION
**Pull request by Riceblade**
hello eznick

### CODE REVIEW

**Review by lelboatz — CHANGES_REQUESTED** [review]
Make sure to change var to let and try to use one register("chat") for all cases unless you are using .setCriteria()

File: build/settings/macros.js [hunk 1]
```diff
@@ -358,10 +358,45 @@ class Macros {
...
+        subcategory: ""
+    })
+    Toggled = "";
+    @TextProperty({
+        name: "Ocelot Level",
+        description: "Enter the level of your ocelot here.",
+        category: "Foraging",
+        placeholder: "100"
+    })
+    Level = "";
```
Comment (lelboatz) [inline build/settings/macros.js:375]: Make sure to use camel case for variables

**Review by lelboatz — CHANGES_REQUESTED** [review]

File: build/features/macros/packetrod.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+let lastSwap = new Date().getTime();
+
+const PacketRod = () => {
+    new Thread(()=>{
+        if (macros.Toggled == true){
+            if (macros.moduleMode == 1) {
+            for (let i = 0; i < 9; i++) {
+                if (Player.getInventory().getStackInSlot(i) !== null && Player.getInventory().getStackInSlot(i).getName().removeFormatting().toLowerCase().includes("rod")) {
+                    if (new Date().getTime() - lastSwap > 500) {
+                        let Axe;
```
Comment (lelboatz) [inline build/features/macros/packetrod.js:12]: use lowercase here

File: build/features/macros/packetrod.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                        let Rod;
```
Comment (lelboatz) [inline build/features/macros/packetrod.js:13]: and here

**Review by lelboatz — CHANGES_REQUESTED** [review]

**Comment by Riceblade** [comment]
done fixing

**Comment by Riceblade** [comment]
🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

File: build/settings/macros.js [hunk 4]
```diff
@@ -358,10 +358,45 @@ class Macros {
...
-
+    
+     @SwitchProperty({
+        name: "Enable",
+        description: "Enable or disable this module",
+        category: "Foraging",
+        subcategory: ""
+    })
+    toggled = false;
+    @TextProperty({
```
Comment (lelboatz) [inline build/settings/macros.js:369]: use slider property instead

File: build/settings/macros.js [hunk 5]
```diff
@@ -358,10 +358,45 @@ class Macros {
...
+        name: "Ocelot Level",
+        description: "Enter the level of your ocelot here.",
+        category: "Foraging",
+        placeholder: "100"
+    })
+    level = "";
+    @TextProperty({
```
Comment (lelboatz) [inline build/settings/macros.js:376]: use slider property instead

File: build/settings/macros.js [hunk 6]
```diff
@@ -358,10 +358,45 @@ class Macros {
...
+        name: "Minimum Rod Delay",
+        description: "",
+        category: "Foraging",
+        placeholder: "300"
+    })
+    min = "";
+    @TextProperty({
```
Comment (lelboatz) [inline build/settings/macros.js:383]: use slider property instead

**Review by lelboatz — CHANGES_REQUESTED** [review]

**Comment by Riceblade** [comment]
awesome possum

**Review by lelboatz — APPROVED** [review]

**Pull request merged**